<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MODBUSRTU驱动配置界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="Tab1">
      <property name="styleSheet">
       <string notr="true">QWidget#widget{position:absolute;
top:10px;
left:20px;}</string>
      </property>
      <attribute name="title">
       <string>数据源配置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8" rowstretch="9,1" columnstretch="3,7">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="Label">
            <property name="text">
             <string>索引:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="index1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>100</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_2">
            <property name="text">
             <string>名称:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="name1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>数据源别名</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_3">
            <property name="text">
             <string>使能:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="enable1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Label_4">
            <property name="text">
             <string>备注:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="desc1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>描述性信息</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="Label_5">
            <property name="text">
             <string>端口号:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="com">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="Label_6">
            <property name="text">
             <string>波特率:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="baudRate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Label_7">
            <property name="text">
             <string>校验方式:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="parity">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据位:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="dataBits">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>停止位:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="stopBits"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>超时时间:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSpinBox" name="responseTime">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>重否次数:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSpinBox" name="numberOfRetries">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="connectTest">
            <property name="text">
             <string>连接测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearDataSource">
            <property name="text">
             <string>清除数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveDataSource">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab2">
      <attribute name="title">
       <string>数据集配置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9" columnstretch="3,7">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_6" rowstretch="4,5,1" columnstretch="1,1">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>读信息:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label1">
               <property name="text">
                <string>主机地址:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="server_read"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>功能码:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="code_read"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>开始地址:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="startAddr_read"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>地址长度:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="length_read"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string> 写信息:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>开始地址:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>功能码:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>地址长度:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="code_write"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>主机地址:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="server_write"/>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="startAddr_write"/>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="length_write"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="getDataTest">
               <property name="text">
                <string>读测试</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="setDataTest">
               <property name="text">
                <string> 写测试</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearDataSet">
               <property name="text">
                <string>清除数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveDataSet">
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="2">
              <widget class="QComboBox" name="dataSource"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>数据源:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLineEdit" name="desc2">
               <property name="placeholderText">
                <string>描述性信息</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="Label_18">
               <property name="text">
                <string>备注:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="Label_17">
               <property name="text">
                <string>使能:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="name2">
               <property name="placeholderText">
                <string>数据集别名</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="index2">
               <property name="minimum">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QCheckBox" name="enable2">
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>允许 写:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="Label_15">
               <property name="text">
                <string>索引:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="Label_16">
               <property name="text">
                <string>名称:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QCheckBox" name="writeEnable">
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView2">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
